### PR TITLE
Story 13

### PR DESCRIPTION
**Pull request by amhursh**
Related Issue: #15 

Summary: Add ability to see past orders and several order models. Build order item relationships.

Author(s): amhursh

### CODE REVIEW

**Comment by JoelLindow** [comment]
This looks great! Good work.

File: app/models/order.rb [hunk 1]
```diff
@@ -1,3 +1,22 @@
 class Order < ApplicationRecord
   belongs_to :user
+  has_many :orders_items
+  has_many :items, through: :orders_items
+
+  enum status: %w(ordered paid cancelled completed)
```
Comment (limsammy) [inline app/models/order.rb:6]: Excited to use this enum :3

File: app/models/order.rb [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+  def total_order_price
+  	items.sum(:price)
+  end
+
+  def item_quantity(item)
+  	id = item.id
+  	items.where(id: id).count
+  end
+
+  def item_subtotal(item)
```
Comment (limsammy) [inline app/models/order.rb:17]: Will begin model testing order soon.

File: app/models/orders_item.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class OrdersItem < ApplicationRecord
+  belongs_to :item
```
Comment (limsammy) [inline app/models/orders_item.rb:2]: Join table looks good

**Review by limsammy — APPROVED** [review]
Looks good. Going to merge.

**Pull request merged**